### DISCUSSION TITLE
is there a ready for use version

### DISCUSSION THREAD

**Discussion by abenzakour** [Laravel-Backpack/community-forum#1154] [Ideas]
Hello, 
I purshased a BackPack pro/Devtools; but I'm not able to install correctly all other plugins ( Language switcher, permissions.....) is there a ready to use version with all packages inside ?

**Comment by karandatwani92** [comment]
Hey @abenzakour 

Thanks for your purchase. Can you share some error info to know exactly which step or add-on you are facing the issue with?

Here are the general steps:
1. Setup a Laravel project
2. Config `.env` and set `APP_URL` to the URL you use to access your application in the browser, for example: 
`http://127.0.0.1:8000` or `https://something.test`
3. Run `php artisan backpack:install`, follow the steps, and choose the purchased items to install. I.e., `Pro`, `devtools`, `Editable Columns`
> Access your dashboard... Try creating a CRUD. Your Backpack setup is supposed to be ready and running with `pro` and `dev tools.` This single command covers what most of the users need.
4. Now continue installing more [Free Official Add-ons](https://backpackforlaravel.com/docs/6.x/add-ons-official) one by one using the steps instructed in their Readme.


Currently, we only have [Demo Repo](https://github.com/Laravel-Backpack/demo), which comes with all other plugins. But:
- Users need to have access to EVERYTHING. I mean, all the paid add-ons
- It is not Empty. It already includes many CRUDs that you may not want.

 I understand what convenience you are looking for, BUT:
 - Every user does not have EVERYTHING purchased
 - Many users do not start with a fresh project.
 - Each user has their own combination of add-on requirements. Everyone does not prefer to install and use everything.
 
Although I find your requirement valid, I will discuss it with @tabacitu and the team.
Let me know if you continue to face the issue.

**Reply by abenzakour** [reply]
Hello @karandatwani92 , 

Thank you for your detailed response. Actually, I have worked with several Laravel CRUD generators before, but I found BackPack to be completely different. It significantly helps deliver projects to clients quickly, thanks to the vast set of predefined features it offers. Moreover, it allows you to fully customize even the most complex elements you may encounter during development, which is truly amazing.

However, my only issue is that I haven't been able to get a version that integrates all the plugins, with French as the default language instead of English, and that groups the critical modules (Backup, Logs, Roles, etc.) under one menu with access restricted to the superadmin.

It would be greatly appreciated, if you offer also a ready to use version with at least users/roles management and translation; meanwhile I will use the demo version and delete the useless things.

Thank you very much @karandatwani92

**Reply by karandatwani92** [reply]
Hey @abenzakour 

I discussed it with the team and finalized adding it to the plans [here](https://github.com/Laravel-Backpack/CRUD/issues/5663). The priority is a little less, but we'll have it in the future.

Thanks for the lovely feature request.
